packages/ppx_relit/ppx_relit.0.1/opam

```diff
@@ -0,0 +1,30 @@
+opam-version: "2.0"
+name: "ppx_relit"
+version: "0.1"
+maintainer: "Charles Chamberlain <charlespipin@gmail.com>"
+authors: [
+  "Charles Chamberlain <charlespipin@gmail.com"
+  "Cyrus Omar <cyrus.omar@gmail.com"
+]
+homepage: "https://github.com/cyrus-/relit"
+bug-reports: "https://github.com/cyrus-/relit/issues"
+license: "MIT"
+dev-repo: "git://github.com/cyrus-/relit.git"
+build: [
+  ["dune" "build" "-p" "ppx_relit"]
+]
+depends: [
+  "dune"
+  "ocamlfind"
+  "ocaml-migrate-parsetree"
+  "relit-reason"
+  "base64"
+  "ppxlib"
+  "base-unix"
+  "ocamlbuild"
```

Comment:
does this still depend on ocamlbuild? Looks like it's been ported to Dune

Reply:
Ah good point, we use ocamlbuild for some tests but it shouldn't be needed to build/use it.

Reply:
If you use it in tests you can still mark it as a test dependency by adding {test} next to it